go.mod

```diff
@@ -10,13 +10,17 @@ require (
 	github.com/kr/pretty v0.1.0 // indirect
 	github.com/pkg/errors v0.9.1
 	github.com/stretchr/testify v1.5.1
-	golang.org/x/net v0.0.0-20200930145003-4acb6c075d10 // indirect
-	golang.org/x/sys v0.0.0-20200930185726-fdedc70b468f // indirect
-	golang.org/x/text v0.3.3 // indirect
+	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9 // indirect
+	golang.org/x/lint v0.0.0-20190313153728-d0100b6bd8b3 // indirect
+	golang.org/x/net v0.0.0-20201216054612-986b41b23924 // indirect
+	golang.org/x/sys v0.0.0-20201218084310-7d0127a74742 // indirect
+	golang.org/x/text v0.3.4 // indirect
+	golang.org/x/tools v0.0.0-20190524140312-2c0ae7006135 // indirect
 	google.golang.org/grpc v1.27.0
 	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
 	gopkg.in/yaml.v2 v2.2.8 // indirect
 	gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c
+	honnef.co/go/tools v0.0.0-20190523083050-ea95bdfd59fc // indirect
```

Comment:
What is honnef for?

Reply:
Looks like an indirect dependency of several modules:

```
jirka@jiria-linux:~/ecp/src/github.com/microsoft/moc-pkg$ go mod graph | grep
honnef
google.golang.org/genproto@v0.0.0-20190425155659-357c62f0e4bb honnef.co/go/tools@v0.0.0-20190102054323-c2f93a96b099
google.golang.org/api@v0.4.0 honnef.co/go/tools@v0.0.0-20190102054323-c2f93a96b099
go.etcd.io/etcd@v0.5.0-alpha.5.0.20201125193152-8a03d2e9614b honnef.co/go/tools@v0.0.0-20190523083050-ea95bdfd59fc
google.golang.org/genproto@v0.0.0-20190819201941-24fa4b261c55 honnef.co/go/tools@v0.0.0-20190102054323-c2f93a96b099
google.golang.org/genproto@v0.0.0-20190307195333-5fe7a883aa19 honnef.co/go/tools@v0.0.0-20190102054323-c2f93a96b099
github.com/microsoft/moc@v0.10.1-alpha.4 honnef.co/go/tools@v0.0.0-20190523083050-ea95bdfd59fc
```